api/src/db/migrations/v123_remove_account_creation_sidecards.test.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  generateV122Business,
+  generateV122Preferences,
+  generateV122UserData,
+} from "./v122_remove_welcome_and_welcomeupandrunning_sidecards";
+import { migrate_v122_to_v123 } from "./v123_remove_account_creation_sidecards";
+
+describe("migrate_v122_to_v123", () => {
+  it("removes the successful-registration card from sidebarCards", () => {
+    const id = "biz-1";
+    const v122Preferences = generateV122Preferences({
+      visibleSidebarCards: ["welcome", "welcome-up-and-running", "other-card", "successful-registration"],
```

Comment:
[pebble] while this technically works / passes as a unit test, I think it's kind of confusing because v122 was to remove "welcome" and "welcome-up-and-running" cards, so a user going into this function should never have those cards in their userData, and it feels weird/misleading to include them in the test setup. We could test pretty much anything else.

Reply:
Ah, you are totally right. I will fix right away